invenio_rdm_records/assets/semantic-ui/js/invenio_rdm_records/src/deposit/customFields/Thesis.js

```diff
@@ -41,26 +43,32 @@ export class Thesis extends Component {
               <label className="helptext mb-0">{university.description}</label>
             )}
           </Grid.Column>
-          <Grid.Column width="6">
-            <Input
-              fieldPath={`${fieldPath}.department`}
-              label={department.label}
-              placeholder={department.placeholder}
-            />
-            {department.description && (
-              <label className="helptext mb-0">{department.description}</label>
-            )}
-          </Grid.Column>
-          <Grid.Column width="4">
-            <Input
-              fieldPath={`${fieldPath}.type`}
-              label={type.label}
-              placeholder={type.placeholder}
-            />
-            {type.description && (
-              <label className="helptext mb-0">{type.description}</label>
-            )}
-          </Grid.Column>
+
+          {department && (
```

Comment:
won't this cause the field to be hidden when we are creating a new record in the deposit form?

Reply:
after a bit of thinking: can you add a comment that this check is for backwards compat? same for the comment below